src/client/components/Options.tsx

```diff
@@ -103,7 +105,7 @@ const OptionsStyled = styled.div`
     box-shadow: inset 0 0 0 1.15px ${Colors.gray_light};
     border-radius: 8px;
     padding: 4px;
-    z-index: 1;
+    z-index: ${OPTIONS_Z_INDEX};
```

Comment:
Why does this work?  Isn't `OPTIONS_Z_INDEX` also 1?

Reply:
So instead of putting the z-index in the tooltip to a constant 2, I'm changing it here to the constant `OPTIONS_Z_INDEX` and setting it in the tooltip to `OPTIONS_Z_INDEX + 1`

Reply:
Oh, I totally missed the bit at the bottom where you styled the tooltip to be + 1.  Thanks for explanation